SYCL/Basic/alloc_pinned_host_memory.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: level_zero || cuda
+// REQUIRES: cuda
```

Comment:
Does the test failing on Level_Zero satbily? If it is I would prefer to mark test as XFAIL: level_zero

Reply:
created separate tests for level_zero and cuda